src/plugins/content_management/common/rpc/msearch.ts

```diff
@@ -22,6 +22,7 @@ export const mSearchSchemas: ProcedureSchemas = {
         }
       ),
       query: searchQuerySchema,
+      options: schema.maybe(schema.object({}, { unknowns: 'allow' })),
```

Comment:
How is this used?

Reply:
I think this has been superseeded by @Dosant request of remove `options` from `mSearch`. I'll remove it.

```suggestion
```